### ISSUE TITLE
Exception thrown when Reference Lines tool is on and the active series is replaced

### ISSUE THREAD

**Issue by jbocce** [OHIF/Viewers#3319]
### Describe the Bug

This bug was logged as a result of this post: https://community.ohif.org/t/referencelinestool-can-cause-functions-such-as-length-and-annotation-to-fail/717

### Steps to Reproduce:

1. Launch the v3 stable demo.
2. Turn on the Reference Lines tool
3. Double click or drag and drop a new series into view.

### The current behavior

The following exception is thrown in the browser console...

```
caught TypeError: Cannot read properties of undefined (reading 'getActiveCamera')
    at Se.getVtkActiveCamera (Viewport.js:381:31)
    at Se.getCamera (Viewport.js:385:32)
    at cb.renderAnnotation (ReferenceLinesTool.js:97:79)
    at triggerAnnotationRender.js:93:43
    at Array.forEach (<anonymous>)
    at triggerAnnotationRender.js:97:26
    at d (draw.js:4:5)
    at Object._triggerRender (triggerAnnotationRender.js:89:9)
    at _renderFlaggedViewports (triggerAnnotationRender.js:17:26)
```
### The expected behavior

No exception is thrown.
> :warning: Reports we cannot reproduce are at risk of being marked stale and
> closed. The more information you can provide, the more likely we are to look
> into and address your issue.

# Another bug
Reference lines gets dashed if their viewport is tracked

**Comment by kundanashish** [comment]
Facing the same issue. Reference line are not working properly.
Any update on this fix.

**Comment by rghgit1** [comment]
@jbocce 
Thanks for opening the issue. Noticed the same error in logs when Reference Lines are not working properly. A fix for this will greatly help.

**Comment by meitingshuoguo** [comment]
I have same issue. please tell me why it is close and how solve it. thanks

**Comment by jbocce** [comment]
@meitingshuoguo I am not able to reproduce this problem on our demo server https://viewer.ohif.org/. If possible could you please outline the steps which reproduce the problem for you? Thanks.

**Comment by meitingshuoguo** [comment]
Thanks for the reply, after my testing, v3.6.0 will have the issue. Specific steps:
1. http://localhost:3000/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095438.5
2. grid layout select 2
3. select reference lines
4. grid layout select 1

> I only tested v3.6.0 and v3.7.0

**Comment by jbocce** [comment]
@meitingshuoguo I tried on the master branch with the steps mentioned but did not see an error/exception.

**Comment by meitingshuoguo** [comment]
For some reason, I'm using a version that branches with tag v3.6.0. It has this problem, not master branch.

**Comment by raviaiken** [comment]
@jbocce You are right, this is working fine on master branch. Any pointers on how to fix this with v3.6.0 or earlier?

**Comment by raviaiken** [comment]
> @raviaiken I have nothing off the top of my head as to how to fix this is in an earlier version. You could start by using `git bisect` to determine which commit fixed the problem and go from there.

Thank you. Changes done as part of v3.7.0-beta.63 (remove viewportIndex and only rely on viewportId) has fixed this issue.
